spec/services/atmosphere/appliance_vms_manager_cep_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+describe Atmosphere::ApplianceVmsManager do
+
+  before(:suite) do
+   Fog.mock!  
```

Comment:
Unnecessary spacing detected.<br>Use 2 (not 1) spaces for indentation.<br>Trailing whitespace detected.